File: code/client/src/app/entities/admin/model/accesorio.ts
import { BaseModel } from '@shared/model';
export class Accesorio extends BaseModel {
  nombre: string | null = null;
  modelo: string | null = null;
  tipo: string | null = null;
  descripcion?: string | null = null;
  codigo_imt: string | null = null;
  precio?: number | null = null;
  url_data_sheet?: string | null = null;
  id_equipo?: number | null = null;
  nombreEquipoAsociado?: string | null = null;
}

File: code/server/src/Core/Entities/Accesorio.cs
using IMT_Reservas.Server.Core.Abstraction;

namespace IMT_Reservas.Server.Core.Entities;

public class Accesorio : Entity
{
    public string Nombre { get; set; } = string.Empty;
    public string? Descripcion { get; set; }
    public string Modelo { get; set; } = string.Empty;
    public string? UrlDataSheet { get; set; }
    public double? Precio { get; set; }
    public int IdEquipo { get; set; }
    public string? Tipo { get; set; }
}
